src/demo/article.html

```diff
@@ -6553,9 +6553,91 @@ <h2>
         References
       </h2>
 
-      <ol>
+      <ol itemtype="https://stencila.github.io/schema/CiteGroup">
         <li>
-          1
+          <ul itemtype="https://stencila.github.io/schema/Cite">
+            <div itemtype="https://schema.org/CreativeWork">
+              <a
+                href="https://doi.org/10.1016/j.cell.2010.07.042"
+                itemprop="title"
+                >Cell flow reorients the axis of planar polarity in the wing
+                epithelium of Drosophila</a
+              >
+
+              <ol itemprop="authors">
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:B+Aigouy%22"
+                    >B Aigouy</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:R+Farhadifar%22"
+                    >R Farhadifar</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:DB+Staple%22"
+                    >DB Staple</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:A+Sagner%22"
+                    >A Sagner</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:JC+R%C3%B6per%22"
+                    >JC Röper</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:F+J%C3%BClicher%22"
+                    >F Jülicher</a
+                  >
+                </li>
+                <li itemtype="https://schema.org/Person">
+                  <a
+                    href="https://scholar.google.com/scholar?q=%22author:S+Eaton%22"
+                    >S Eaton</a
+                  >
+                </li>
+              </ol>
+
+              <span itemprop="datePublished">(2010)</span>
```

Comment:
Did we decide not to use the `date` tag?

Reply:
Thanks for catching this. I think we should use the `date` tag when applicable. Updated it.